src/sql/src/pure.rs

```diff
@@ -170,6 +132,62 @@ pub async fn purify(mut stmt: Statement<Raw>) -> Result<Statement<Raw>, anyhow::
     Ok(stmt)
 }
 
+async fn purify_postgres_connector(
+    conn: &str,
+    namespace: &str,
+    table: &mut PgTable<Raw>,
+) -> Result<(), anyhow::Error> {
+    let fetched_columns =
+        postgres_util::fetch_columns(conn, namespace, &table.name.to_ast_string()).await?;
```

Comment:
This isn't correct anymore. `table.name` could be something like `db.schema.table` which will be converted to a single string which will break the query in `fetch_columns`.

Reply:
Agreed! Sorry, was pushing too quickly to take advantage of CI. I think I added a sufficient test case for this [here](https://github.com/MaterializeInc/materialize/pull/6299/files#diff-545df3ab0446c06044792bf7899eddd603209203e14e327b12549ec0b0d0350aR23).